src/cclib/parser/cjsonHolder.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of cclib (http://cclib.github.io), a library for parsing
+# and interpreting the results of computational chemistry packages.
+#
+# Copyright (C) 2007-2014, the cclib development team
+#
+# The library is free software, distributed under the terms of
+# the GNU Lesser General Public version 2.1 or later. You should have
+# received a copy of the license along with cclib. You can also access
+# the full license online at http://www.gnu.org/copyleft/lgpl.html.
+
+from .data import ccData
+
+class cjsonHolder:
```

Comment:
This is not exactly what I had in mind. I was thinking of a simple data structure such as a dictionary, which would hold just a mapping from cclib attribute name to CJSON entry.